client-app/ui-kit/components/molecules/button/vc-button.vue

```diff
@@ -107,25 +108,41 @@ const componentTag = computed(() => {
   if (isRouterLink.value) {
     return "router-link";
   }
+
   if (isExternalLink.value) {
     return "a";
   }
+
   return "button";
 });
 
-const linkAttr = computed(() => {
+const attrs = computed(() => {
   if (componentTag.value === "router-link") {
     return { to: props.to };
   }
+
   if (componentTag.value === "a") {
     return { href: props.externalLink };
   }
+
+  if (props.minWidth) {
+    const minWidth = typeof props.minWidth === "number" ? `${props.minWidth}px` : props.minWidth;
+
+    return {
+      style: {
+        minWidth,
+      },
+    };
+  }
```

Comment:
I think number here is redundand as we can simply pass "120px" when need to use pixels